easybuild/easyconfigs/m/METIS/METIS-5.1.0-intel-2015b-32bitIDX.eb

```diff
@@ -0,0 +1,24 @@
+name = 'METIS'
+version = '5.1.0'
+#default 32-bit IDTYPEWIDTH, no patch used
+versionsuffix = '-32bitIDX'
+
+homepage = 'http://glaros.dtc.umn.edu/gkhome/metis/metis/overview'
+description = """METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes,
+and producing fill reducing orderings for sparse matrices. The algorithms implemented in METIS are based on the
+multilevel recursive-bisection, multilevel k-way, and multi-constraint partitioning schemes."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis',
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/OLD',
+]
+
+builddependencies = [('CMake', '3.3.2')]
+
+configopts = [' ', 'shared=1 ']
```

Comment:
@hajgato: keeping both/all easyconfigs (of the same version) in sync makes sense, no?
Only do it for 5.1.0 though, not in all existing METIS easyconfigs.

Also, the space at the end of each value shouldn't be needed?
If fact, it's probably wrong, since the METIS easyblock checks for `shared=1` (without a space at the end) to define `self.lib_exts`, which is used in the sanity check (so, the sanity check is not checking for the `.so` if you include the space).